scss/_patterns_breadcrumbs.scss

```diff
@@ -7,15 +7,16 @@
     margin: 0;
     padding: 0;
     width: 100%;
+  }
 
-    &__items {
-      margin-left: 0;
-      padding-left: 0;
-    }
+  .p-breadcrumbs__items {
+    margin-bottom: $spv--large;
```

Comment:
I'm not sure if this is needed. If we have margin from small caps text on the `__item` this probably should not be needed.

Reply:
The margin from small caps is overriden by `    margin-bottom: $spv-nudge-compensation;` which I'm reluctant to touch because it sounds like some baseline grid magic

Reply:
Ah sorry I was confused what your previous comment referred to!

Reply:
Sure, I guess it used to be a margin to align with baseline, but because we are replacing margin completely, changing font text style to something else etc, old styles need to be removed, so we can start with clean state, and then see if they baseline needs adjustments.

But if we use defaults from small caps we should not need anything else (as that should be aligned already).